### DISCUSSION TITLE
App Icon

### DISCUSSION THREAD

**Discussion by gdreelin** [FlareSolverr/FlareSolverr#112] [General]
I was able to add this through the CA in UnRAID and it worked great. I wanted to add a request for the future of adding an icon to the app so it shows up with one in UnRAID or whatever you add it too instead of boring nothing icon. Just a thought....thanks

**Comment by ngosang** [comment]
First of all, I know nothing about UnRAID.
What are the requirements? Is a favicon.ico enough? Do you have an example of other Apps / Packages with icon? Could you share some screenshots?

**Comment by gdreelin** [comment]
Sorry UnRAID is just a storage system that allows docker containers to be used. If you got to the App Store in UnRAID you can pull FlareSolver down to make a container. It works great. Just for aesthetic it would be nice to have some icon set in the container. Below is what UnRAID docker page look like and where the link is added to put an icon. I did a test to put an icon there and it worked but it was just a sample icon. I was looking for something that represents FlareSolver itself. It is no biggie just posted as a suggestion. 

![Untitled](https://user-images.githubusercontent.com/802649/115431442-4e5fba80-a1d3-11eb-87bd-2e94ab85871e.png)
![Untitled1](https://user-images.githubusercontent.com/802649/115431450-515aab00-a1d3-11eb-9e25-d1f35c342868.png)

**Comment by ngosang** [comment]
Maybe you can use the official logo => https://github.com/FlareSolverr/FlareSolverr/blob/master/resources/flaresolverr_logo.png

**Reply by gdreelin** [reply]
Thanks that worked like a champ.

**Reply by ngosang** [reply]
Direct link => https://raw.githubusercontent.com/FlareSolverr/FlareSolverr/master/resources/flaresolverr_logo.png
